cpp/daal/src/algorithms/dtrees/dtrees_feature_type_helper.i

```diff
@@ -293,6 +348,68 @@ services::Status ColIndexTaskBins<IndexType, algorithmFPType, cpu>::makeIndex(Nu
     return assignIndexAccordingToBins(entry, aRes, nBins, nRows);
 }
 
+template <typename IndexType, typename algorithmFPType, CpuType cpu>
+services::Status ColIndexTaskBins<IndexType, algorithmFPType, cpu>::makeIndexAverages(NumericTable & nt, IndexedFeatures::FeatureEntry & entry,
+                                                                                      IndexType * aRes, size_t iCol, size_t nRows)
+{
+    std::cout << "Hello from makeIndexAverages" << std::endl;
+    const typename super::FeatureIdx * index = this->_index.get();
+
+    size_t nBins            = 0;
+    size_t i                = 0;
+    algorithmFPType binSize = (index[nRows - 1].key - index[0].key) / _prm.maxBins;
```

Comment:
No, because we never create the binning task when maxBins=0 is selected. Nevertheless, I have added a DAAL_ASSERT for debug later in case this gets changed